### PR TITLE
Manage your timesheet with the calender and sheets

### PR DESCRIPTION
**Pull request by duizendstra**

### CODE REVIEW

File: calendar-sheet-timesheet/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: Manage your timesheet with the calender and sheets
```
Comment (asrivas) [inline calendar-sheet-timesheet/README.md]: I like the title you have in your Medium article but let's go a little shorter:
"Record time and activities in Calendar and Sheets"
or maybe:
"Sync your time tracker to a Calendar and Sheet"

There's already a 'timesheets' solution so I'd like rephrase

File: calendar-sheet-timesheet/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+update_time: 2019-09-08
+---
+
+# Manage your timesheet with the calender and sheets
+
+_Create calender events for your activities, and synchronise to your timesheet in sheets for reporting._
+
+Contributed by Jasper Duizendstra, follow me at [@Duizendstra](https://twitter.com/duizendstra)
+
+Keeping track of the time spent on project for customers can be a challange. Not all time writing apps are user friendly and sometimes you need to record your time in multiple systems. You can end up spending a lot of time writing time on a daily basis in a stubborn system and synchronise that time to other systems.
```
Comment (asrivas) [inline calendar-sheet-timesheet/README.md]: can you make this wrap at 100 characters?

File: calendar-sheet-timesheet/src/Code.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+
+/** 
+ * Please add your solution here if you are housing your implementation in this repo. If you 
+ * are simply linking your code from another repo, please delete this file. 
+ */
+
+function onOpen() {
+    "use strict";
+    var menuEntries = [{
+            name: "run",
```
Comment (asrivas) [inline calendar-sheet-timesheet/src/Code.js]: Can you rename this to be more descriptive? 
"Run TimeTracker"?

File: calendar-sheet-timesheet/src/Code.js [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+            functionName: 'run'
+        }],
+        activeSheet;
+
+    activeSheet = SpreadsheetApp.getActiveSpreadsheet();
+    activeSheet.addMenu('myTime', menuEntries);
+
+}
+
+function run() {
```
Comment (asrivas) [inline calendar-sheet-timesheet/src/Code.js]: can you add method comments?

File: calendar-sheet-timesheet/src/Code.js [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+    }
+
+    mainSpreadsheet = SpreadsheetApp.openById(mainSpreadSheetId);
+    hourSheet = mainSpreadsheet.getSheetByName("Hours");
+    codesSheet = mainSpreadsheet.getSheetByName("Codes");
+
+    return Object.freeze({
+        run: run
+    });
+}
```
Comment (asrivas) [inline calendar-sheet-timesheet/src/Code.js]: nit: newline

File: calendar-sheet-timesheet/src/appsscript.json [hunk 6]
```diff
@@ -0,0 +1,6 @@
+{
+  "timeZone": "YOUR TIMEZONE HERE",
```
Comment (asrivas) [inline calendar-sheet-timesheet/src/appsscript.json]: Just replace this placeholder with your timezone

File: calendar-sheet-timesheet/src/Code.js [hunk 7]
```diff
@@ -0,0 +1,211 @@
...
+                if (currentIds.indexOf(id) === -1 && id !== "") {
+                    hourSheet.deleteRow(index + 1);
+                }
+            }
+        });
+
+        formatSheet();
+        SpreadsheetApp.flush();
+        SpreadsheetApp.setActiveSheet(hourSheet);
+
```
Comment (asrivas) [inline calendar-sheet-timesheet/src/Code.js]: keep only 1 space in between code sections

File: calendar-sheet-timesheet/src/Code.js [hunk 8]
```diff
@@ -0,0 +1,211 @@
...
+
+        console.log("Finished processing hours.");
+
```
Comment (asrivas) [inline calendar-sheet-timesheet/src/Code.js]: remove last new line

File: calendar-sheet-timesheet/src/Code.js [hunk 9]
```diff
@@ -0,0 +1,211 @@
...
+                    row: index + 1,
+                    value: entry
+                };
+                return true;
+            }
+        });
+        return dataRow;
+    }
+
+    function handleNewEvent(event) {
```
Comment (asrivas) [inline calendar-sheet-timesheet/src/Code.js]: can you check the alignment on these?

**Comment by duizendstra** [comment]
I made the changes. I checked, my email is on the https://cla.developers.google.com/clas. It is an G Suite address, that might be an issue?

**Comment by asrivas** [comment]
cla: yes
Personally verified

**Pull request merged**